source/crud/tests/v2/aggregate-out-readConcern.yml

```diff
@@ -0,0 +1,145 @@
+data:
+    - { _id: 1, x: 11 }
+    - { _id: 2, x: 22 }
+    - { _id: 3, x: 33 }
+
+minServerVersion: '4.1'
+
+collection_name: &collection_name 'test_aggregate_out_readconcern'
+
+tests:
+  -
+    description: "readConcern majority with out stage"
+    operations:
+      -
+        object: collection
+        name: aggregate
+        collectionOptions:
+          readConcern: { level: "majority" }
+        arguments:
```

Comment:
Would it make sense to `&arguments` here instead of just `&output_collection`? I think the pipeline is the same for each test.